core-js/src/main/javascript/cdf/dashboard/Utils.js

```diff
@@ -734,6 +734,52 @@ define([
     return "rgb(" + rgb.join(",") + ")";
   };
 
+  /**
+   * Determines if `path` ends with `action`,
+   * immediately preceded by a `\` or `/` separator character.
+   *
+   * Pattern to test: `<path> === <basepath>[\/]<action>`.
+   *
+   * This implementation, in particular, strives to not allocate memory.
+   *
+   * @param {*} path - The path.
+   * @param {*} action - The action.
+   * @return {boolean} `true` if yes; `false`, otherwise.
+   */
+  Utils.pathIncludesAction = function (path, action) {
+
+    if(typeof path !== "string" || typeof action !== "string") {
+      return false;
+    }
+
+    if(action.length >= path.length) {
```

Comment:
Why are you excluding the case where they are equal? `action` is actually included in `path` in that case — a legitimate use case imo.

Otherwise, this is an optimization which you can be simply delegated to `endsWith`.

Reply:
It was a mistake, i was thinking only in case where path includes action.